app/views/works/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<body>
```

Comment:
This isn't how to use the `<body>` tag. The body tag is meant to contain all of the content HTML (in direct opposition to the `<head>` tag, which contains all the meta data about the site). A `<body>` tag only occurs once in a given HTML document.

Your application.html.erb layout already has a `<body>` tag, so you don't need to add it here.